src/main/java/org/codinjutsu/tools/jenkins/model/BuildStatusEnum.java

```diff
@@ -37,7 +37,8 @@
     NULL("Null"),
     // TODO: handle the folder-case explicitly. @mcmics: use better Folder Detection
     // instead of simply making it a BuildStatusEnum so that the icon renders
-    FOLDER("Folder");
+    FOLDER("Folder"),
+    RUNNING("Running", Color.GRAY);
```

Comment:
Your reasoning was cut off so can't tell why you would reuse an existing state, but in my opinion the state isn't null so that state shouldn't be used. Using the state from the last build won't work when aggregating multiple builds.
If at any point in time the icon for running should change I think it's better to be able by changing this one line (and apparently the renderer) than to refactor setting another state.

I added the icon in the BuildStatusRenderer as well. I'm not entirely sure why that class lists all the enum options instead of iterating over them though.

Reply:
Could you add also to [DefaultBuildStatusEnumRenderer](https://github.com/nvdweem/jenkins-control-plugin/blob/af4ce59dfe7527d9289613aa8fee87ffc216ab90/src/main/java/org/codinjutsu/tools/jenkins/view/DefaultBuildStatusEnumRenderer.java#L22) and both tests for the renderer